README.md

```diff
@@ -1 +1,5 @@
-# MathMagicians
\ No newline at end of file
+# MathMagicians
+
+```
+Linters and react setup from create-react-app
+```
```

Comment:
- [ ] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project.